websites/P/PonyTown/presence.ts

```diff
@@ -0,0 +1,116 @@
+const presence = new Presence({
+  clientId: "748780200006778892"
+});
+
+const selectors = {
```

Comment:
Codacy found an issue: [Combine this with the previous 'const' statement.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113511)